EKF/ekf.cpp

```diff
@@ -88,6 +88,8 @@ bool Ekf::init(uint64_t timestamp)
 
 	_dt_ekf_avg = FILTER_UPDATE_PERIOD_S;
 
+	_ang_rate_delayed_raw.zero();
```

Comment:
This is not necessary, as the matrix::Vector will be initialized with zeros anyway. Also for other variables we left out an explicit assignment to zero in the `init()` function.